sdk/core/azure-json-gson/pom.xml

```diff
@@ -52,6 +52,7 @@
     <javadoc.excludePackageNames/>
     <javadoc.sourcepath>${basedir}/src/main/java</javadoc.sourcepath>
 
+    <checkstyle.suppressionsLocation>checkstyle-suppressions.xml</checkstyle.suppressionsLocation>
   </properties>
```

Comment:
No SpotBugs exclusion file for this package?

Reply:
This doesn't ship so I left Spotbugs disabled for now